src/Profiles.ts

```diff
@@ -315,20 +324,13 @@ export class Profiles { // Processing stops if there are no profiles detected
             value: userName
         };
         userName = await vscode.window.showInputBox(options);
-        return userName;
-    }
-
-    private async rempromptPassword() {
-        let passWord: string;
-        let options: vscode.InputBoxOptions;
         options = {
             placeHolder: localize("promptcredentials.option.prompt.password.placeholder", "Password"),
             prompt: localize("promptcredentials.option.prompt.password", "Enter a password for the connection"),
             password: true,
             value: passWord
         };
         passWord = await vscode.window.showInputBox(options);
-        return passWord;
```

Comment:
I was unsure about calling the `createBasicZosmfSession` because it would be code-duplication and because I am unsure if we want to just store the credentials in the profile without creating the session.

Reply:
You will probably need this when updating a profile:

await new CliProfileManager({
        ...
    }).update

Reply:
Yeah updated the code.